src/lib/select/select.ts

```diff
@@ -360,6 +360,9 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
     this._multiple = coerceBooleanProperty(value);
   }
 
+  /** Whether to center the active option over the trigger. */
+  @Input () disableOptionCentering: boolean = false;
```

Comment:
Remove the extra space before `@Input`.

Reply:
I've updated it along with the changes @willshowell mentioned. It changes the way we use the new option so the unit test template has been adjusted accordingly.